beacon-chain/state/v1/state_trie_test.go

```diff
@@ -66,49 +66,6 @@ func TestInitializeFromProto(t *testing.T) {
 	}
 }
 
-func TestInitializeFromProtoUnsafe(t *testing.T) {
```

Comment:
Duplicate test no longer needed after merging `InitializeFromProto` and `InitializeFromProtoUnsafe`.